source/assets/js/helpers/module.js

```diff
@@ -12,7 +12,7 @@ class EstaticoModule {
 	* @param  {object} options - The options passed as data attribute in the Module
 	*/
 	constructor($element, _defaultData, _defaultOptions, data, options) {
-		this.name = this.constructor.name.toLowerCase();
```

Comment:
See discussion in https://github.com/unic/estatico/pull/32
@orioltf, we should sit together and have a look at this.